python/ql/src/semmle/python/objects/ObjectAPI.qll

```diff
@@ -684,6 +705,15 @@ module ClassValue {
         result = TBuiltinClassObject(Builtin::special("unicode"))
     }
 
+    /** Get the `ClassValue` for the `str` class. This is `bytes` in Python 2,
+    and `unicode` in Python 3. */
```

Comment:
Is this what you meant? :blush: 

```
/** Get the `ClassValue` for the `str` class. This is `ClassValue::bytes` for Python 2,
    and `ClassValue::unicode` for Python 3. */
```

With the current comment, I read it as returning the result of the _python code_ `bytes` for Python 2, and `unicode` for Python 3 (which makes no sense)

Reply:
Ah, indeed `unicode` does not make sense. It is intended to return the class corresponding to what you get when you write `str` in Python. In Python 2, this is indeed the same as `bytes` (that is, `str is bytes` is true), and in Python 3 it's... `str`.

I must admit, I just assumed the corresponding comment for `ClassObject theStrType()` in `ClassObject.qll` was correct. (It suffers from the same error.)